component/admin/libraries/categoryClass.php

```diff
@@ -55,6 +55,7 @@ public static function categoriesTree()
 				}
 				$v->level -= 1;
 				$pt       = $v->parent;
+				if (array_key_exists($pt, $children) === false) continue;
```

Comment:
We can't do a continue here since we would then not get the category hierarchy.
The correct code replacement should be 

`$list     = array_key_exists($pt, $children) ? $children[$pt] : array();`

Reply:
PR updated with this change.